### ISSUE TITLE
A custom fg for the URL while in `insert mode`

### ISSUE THREAD

**Issue by Pound-Hash** [qutebrowser/qutebrowser#7466]
We have:
```
colors.statusbar.url.error.fgForeground 
colors.statusbar.url.fgDefault
colors.statusbar.url.hover.fgForeground  
colors.statusbar.url.success.http.fgForeground 
colors.statusbar.url.success.https.fgForeground   
colors.statusbar.url.warn.fgForeground  
```

Would be great to have something like **colors.statusbar.url.InsertMode**.

**Comment by The-Compiler** [comment]
Can you elaborate on why? To convey the same information as before, those 6 settings would need to be duplicated for every mode, which is pretty much a no-go.

**Comment by Pound-Hash** [comment]
> Can you elaborate on why? To convey the same information as before, those 6 settings would need to be duplicated for every mode, which is pretty much a no-go.

Did I ask about this before? Sorry if I did; I don't recall.

The issue is that it's difficult to have a legible URL contrast with the different status bar colors: normal, insert mode, caret mode, passthrough....

In those modes, their indicating text can be appropriately given a contrasting color, e.g., the text INSERT MODE while in insert mode. But the URL is always the same color (so long as the page loads).

Here are screenshots of what I've come up with, but I would hope we could gain more control of the URLs string:

![image](https://user-images.githubusercontent.com/51193876/200192871-11022217-8134-4b36-99e3-03080893f062.png)
![image](https://user-images.githubusercontent.com/51193876/200192883-48249fe6-6faf-4adf-a8dc-99ea9fabeda8.png)
![image](https://user-images.githubusercontent.com/51193876/200192894-4d12e007-fcb4-4fc8-bceb-e5bdbc76a730.png)

**Comment by The-Compiler** [comment]
I don't think you did!

I'll need to think about it for a bit. While I do have the same issue (well, way worse actually!) with my colorscheme:

![image](https://user-images.githubusercontent.com/625793/200193067-071ae584-38da-4ebf-9972-39956b0e395c.png)

![image](https://user-images.githubusercontent.com/625793/200193077-cc6de73c-83dc-4261-8089-900a84f9af04.png)

I do find it problematic that the information that the current URL colors convey would get lost... Maybe it'd work as an override, then at least it could be set to `None` by default.

**Comment by Pound-Hash** [comment]
Yeah, I'm colorblind and it's difficult to get colors to contrast enough with my color schemes. I have to use minimal setups. It would be really beneficial for me to have more control over all things color.
